_redirects/boards-mpfs-sev-kit-sev-kit-user-guide.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: forward
+permalink: /redirects/boards-mpfs-sev-kit-es
```

Comment:
@praveenkumari30718 Shouldn't "boards-mpfs-sev-kit-es" be "boards-mpfs-sev-kit-es-sev-kit-user-guide"?

If so, then the same goes for targetname: and targettitle:

Reply:
permalink should also be boards-mpfs-sev-kit-es-sev-kit-user-guide

i.e. 
permalink: /redirects/boards-mpfs-sev-kit-es-sev-kit-user-guide